sphinxcontrib/redoc.py

```diff
@@ -134,17 +134,16 @@ def assets(app, exception):
     # need to either ensure its existence here or do not try to copy  assets
     # in case of failure.
     if not exception:
-        copyfile(
-            os.path.join(_HERE, 'redoc.js'),
-            os.path.join(app.builder.outdir, '_static', 'redoc.js'))
+        build_dir = os.path.join(app.builder.outdir, '_static')
+        redoc_js = os.path.join(build_dir, 'redoc.js')
+        os.makedirs(build_dir, exist_ok=True)
+        copyfile(os.path.join(_HERE, 'redoc.js'), redoc_js)
```

Comment:
Can you please simply replace `copyfile` in original code with `copy_asset` from Sphinx utils? It will ensure the destination exists on copy.

```python
from sphinx.util.fileutil import copy_asset

copy_asset(
    os.path.join(_HERE, 'redoc.js'),
    os.path.join(app.builder.outdir, '_static'))
```